docs/source/tutorials/ScadaInterfaceExample.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
Usually we put these notebooks in the notebooks folder and add a link to that notebook in the tutorials, would it make sense to do the same here?

Reply:
Ah okay, what is the difference between the notebooks in "tutorials" and "notebooks" then? I was not sure where to put it.

Reply:
The notebooks-folder is convenient because also average analysts might realize that one can actually run/download those.

The tutorials subfolder is quite hidden and only used for the documentation.